.github/workflows/elixir.yml

```diff
@@ -15,43 +15,60 @@ jobs:
     strategy:
       matrix:
         include:
-          - pair:
-              elixir: 1.7
-              otp: 20.0
-          - pair:
-              elixir: 1.8
-              otp: 20.0
-          - pair:
-              elixir: 1.8
-              otp: 21.0
-          - pair:
-              elixir: 1.9
-              otp: 21.0
-          - pair:
-              elixir: 1.9
-              otp: 22.0
-          - pair:
-              elixir: 1.10.3
-              otp: 22.0
-          - pair:
-              elixir: 1.10.3
-              otp: 23.0
+          - elixir: '1.7'
+            otp: '20'
+          - elixir: '1.7'
+            otp: '21'
+          - elixir: '1.7'
+            otp: '22'
+          - elixir: '1.8'
+            otp: '20'
+          - elixir: '1.8'
+            otp: '21'
+          - elixir: '1.8'
+            otp: '22'
+          - elixir: '1.9'
+            otp: '20'
+          - elixir: '1.9'
+            otp: '21'
+          - elixir: '1.9'
+            otp: '22'
+          - elixir: '1.10'
+            otp: '21'
+          - elixir: '1.10'
+            otp: '22'
+          - elixir: '1.10'
+            otp: '23'
+          - elixir: '1.11'
+            otp: '21'
+          - elixir: '1.11'
+            otp: '22'
+          - elixir: '1.11'
+            otp: '23'
+          - elixir: '1.11'
+            otp: '24'
+          - elixir: '1.12'
+            otp: '22'
+          - elixir: '1.12'
+            otp: '23'
+          - elixir: '1.12'
+            otp: '24'
```

Comment:
For the record, this matrix was computed based on https://hexdocs.pm/elixir/1.12/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp.